### PR TITLE
Disable parallel thread for tridesclous, moutainsort4 and herdingspikes

### PR DESCRIPTION
**Pull request by samuelgarcia**

### CODE REVIEW

File: spikesorters/basesorter.py [hunk 1]
```diff
@@ -57,6 +58,9 @@ def __init__(self, recording=None, output_folder=None, verbose=False,
         self.parallel = parallel
         self.params = self.default_params()
 
+        if self.parallel:
+            assert self.have_parallel_thread, 'this sorter do not accept parallel=True for groups'
```
Comment (alejoe91) [inline spikesorters/basesorter.py:62]: Instead of raising I would print a warning and set parallel to False

Comment (samuelgarcia) [reply spikesorters/basesorter.py:62]: No I think it have to fail because it do not work.
And I don't this we should create a custom error for this.

Comment (samuelgarcia) [reply spikesorters/basesorter.py:62]: it is a feature that some sorter don't have so fail, no ?

File: spikesorters/basesorter.py [hunk 2]
```diff
@@ -57,6 +58,9 @@ def __init__(self, recording=None, output_folder=None, verbose=False,
         self.parallel = parallel
         self.params = self.default_params()
 
+        if self.parallel:
+            assert self.compatile_with_parallel_thread, 'this sorter do not accept parallel=True for groups'
```
Comment (alejoe91) [inline spikesorters/basesorter.py:62]: do -> does
Also this is not only for groups, but more generally for a gropuing_property

**Comment by samuelgarcia** [comment]
I htink this patch remove the fail for MS4 ahd HS. No ?

File: spikesorters/basesorter.py (hunk 2 above)
Comment (alejoe91) [reply spikesorters/basesorter.py]: @samuelgarcia can you change this?

Comment (samuelgarcia) [reply spikesorters/basesorter.py]: I did it already. No ?

Comment (alejoe91) [reply spikesorters/basesorter.py]: Yes! Sorry I missed it. We can merge then :)

**Comment by samuelgarcia** [comment]
Can I merge this ?
@colehurwitz  : can I close the # 68 ?

**Comment by alejoe91** [comment]
Yes yes merge please :)

**Pull request merged**